File: scripts/fix_sobre.js
const fs = require('fs');
let content = fs.readFileSync('sobre.html', 'utf8');
content = content.replace(/duration-500/g, 'duration-300');
content = content.replace(/text-white/g, 'text-zinc-50');
content = content.replace(/firecam-logo-web\.png/g, 'firecam-logo-web.webp');
fs.writeFileSync('sobre.html', content);

File: scripts/fix_sobre.py
import sys

with open('sobre.html', 'r', encoding='utf-8') as f:
    content = f.read()

content = content.replace('duration-500', 'duration-300')
content = content.replace('text-white', 'text-zinc-50')
content = content.replace('firecam-logo-web.png', 'firecam-logo-web.webp')

with open('sobre.html', 'w', encoding='utf-8') as f:
    f.write(content)
